Detector/DetCRD/compact/CRD_common_v01/SIT_SimplePlanar_v01_01.xml

```diff
@@ -25,7 +25,7 @@
       <!-- database : sit_simple_planar_sensors_03 -->
       
       <!-- SQL command: "select * from extended_reconstruction_parameters;"  -->
-      <reconstruction strip_width="0." strip_length="92*mm" strip_pitch="0." strip_angle="0*deg"  />
+      <reconstruction strip_width="0." strip_length="92*mm" strip_pitch="0." strip_angle="7*deg"  />
```

Comment:
Is strip_width=0 means the strip do not have width?

Reply:
@ihepzhangyao 
strip_width=0 means it does not defined in simulation, and then it can be defined freely in reconstruction.
Currently strip width and strip pitch are not used in tracking. While new algorithm want to use them, I think the following method will be a good way.
`If(strip_width==0) strip_width=$other definition source$`;
If the design is frozen, their values will be set in future. And then the code will also work well.